x-pack/plugins/fleet/server/services/epm/packages/cache.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AsyncLocalStorage } from 'async_hooks';
+
+import LRUCache from 'lru-cache';
+
+import type { AssetsMap } from '../../../../common/types';
+
+import type { PackageInfo } from '../../../../common';
+
+const cacheStore = new AsyncLocalStorage<CacheSession>();
+
+const PACKAGE_INFO_CACHE_SIZE = 20;
+const PACKAGE_ASSETS_MAP_CACHE_SIZE = 1;
```

Comment:
Should we make those key configurable through kibana config under `fleet.internal.*`

Reply:
We can start without making this configurable, those size are pretty small it should not have a huge impact